README.md

```diff
@@ -153,18 +153,18 @@ Commands:
   No need to initialize a project before using this command.
 
 
-  transact|tx [options] <priv_key> <data> <address> <fee> <gas_limit>
+  transact|tx [options] <priv_key> <data> <address> <gas_limit>
   (Unsafe!) Do regular transaction to a contract, you need pass the private key of sender and address of contract
   you need to initialize a project before using this command.
 
 
-  bond|bnd [options] <public_key> <amount> <fee> <priv_key>
-  (safe) Do Bond transaction, you need pass the validator publickey, stake amount, transaction fee, and private key of sender
+  bond|bnd [options] <public_key> <amount> <priv_key>
```

Comment:
Why did you remove fee from these txs?

Reply:
I think @ToniyaSundaram removed it, because currently intergallactic don't support fee feature yet

Reply:
@b00f  Yes it is in accordance with intergallactic

Reply:
`fee = OutAmount - InAMount.`
 I think gnet should handle it. not intergallactic.